src/main/java/org/gridsuite/modification/server/entities/equipment/modification/attribute/BooleanEquipmentAttributeModificationEntity.java

```diff
@@ -24,7 +24,11 @@
 @Table(name = "booleanEquipmentAttributeModification")
 @PrimaryKeyJoinColumn(foreignKey = @ForeignKey(name = "boolean_equipment_attribute_modification_id_fk_constraint"))
 public class BooleanEquipmentAttributeModificationEntity extends EquipmentAttributeModificationEntity<Boolean> {
-    public BooleanEquipmentAttributeModificationEntity(String equipmentId, String attributeName, Boolean attributeValue, IdentifiableType equipmentType) {
-        super(equipmentId, attributeName, attributeValue, equipmentType);
+
+    public BooleanEquipmentAttributeModificationEntity(EquipmentAttributeModificationInfos equipmentAttributeModificationInfos) {
+        super(equipmentAttributeModificationInfos.getEquipmentId(),
```

Comment:
You can't implement the constructor from DTO with this class because of the generic variable ?

Reply:
?